src/adservice/README.md

```diff
@@ -5,8 +5,8 @@ are provided then it returns random ads.
 
 ## Building Locally
 
-The Ad service uses gradlew to compile/install/distribute. Gradle wrapper is
-already part of the source code. To build Ad Service, run:
+The Ad service requires at least JDK 17 to build and uses gradlew to compile/install/distribute. 
```

Comment:
The markdownlint failed because of:
```
./src/adservice/README.md:8:97 MD009/no-trailing-spaces Trailing spaces [Expected: 0 or 2; Actual: 1]
./src/adservice/README.md:8:81 MD013/line-length Line length [Expected: 80; Actual: 97]
```

Reply:
```
Trailing space found at Ln:8 b'The Ad service requires at least JDK 17 to build and uses gradlew to compile/install/distribute. '
```

You are also good remove the empty space in the end of the line